docs/contribute/code.md

```diff
@@ -0,0 +1,131 @@
+
+### Add stories
+
+Adding a story or set of stories to an example app is a great way to test your work. If you're modifying part of Storybook's core, or one of the addons, there's probably a set of stories you can add to the [`official-storybook`](../../examples/official-storybook). If you're modifying something related to a specific framework, they have their own examples in the monorepo. For instance, [`examples/vue-kitchen-sink`](../../examples/vue-kitchen-sink) is a natural place to add stories for `@storybook/vue`, [`examples/angular-cli`](../../examples/angular-cli) for `@storybook/angular`, and so on.
+
+### Add tests
+
+Unit tests are also part of our test strategy. Therefore, we encourage you to use the following naming convention:
+
+```
++-- parentFolder
+|   +-- [filename].ts
+|   +-- [filename].test.ts
+```
+
+## Submit your contribution
+
+Before submitting your contribution, we recommend that you run the test suite one last time with:
+
+```sh
+yarn test
+```
+
+<div class="aside">
+💡  Storybook uses <a href="https://jestjs.io/"><code>jest</code></a> as part of the testing suite, if you notice that the snapshot tests fail you can re-run and update them with <code>yarn test --update</code>.
+</div>
+
+Doing this prevents last-minute bugs and is also a great way to get your contribution merged faster once you submit your pull request. Failing to do so will lead to one of the maintainers mark the pull request with the **Work in Progress** label until all tests pass.
+
+Once the test suite finishes, it's time to commit, push and open a pull request against Storybook's `next` (default) branch. This branch is where all active development happens and is associated with the latest prerelease version (e.g., `6.3.0-alpha.25`).
+
+If your contribution focus on a bugfix and you want it featured in the next stable release, mention it in the pull request description. We'll try to get it in if it appears to be non-disruptive and fixes a critical bug.
+
+## Working with reproductions
+
+We strongly encourage contributors to create reproductions for their issues. In the same way, it's possible to [develop interactively](#start-developing) against example projects in the monorepo; it's also possible to develop against a reproduction repository.
+
+To do so, run the following command in the root of the monorepo:
+
+```sh
+npx sb@next link https://github.com/your-username/your-project.git
+```
+
+This command creates a project `../storybook-repros/your-project`, and automatically links it to your local Storybook code. After connecting it, you should be able to run Storybook and develop as mentioned [above](#start-developing).
+
+If you already have a reproduction on your local machine, you can similarly link it to your monorepo dev setup with the `--local` flag:
+
+```sh
+npx sb@next link /path/to/local-repro-directory
```

Comment:
Should this use the `--local` flag somewhere?

Reply:
Oops typo 🤦